src/inet/InetLayer.cpp

```diff
@@ -975,18 +975,13 @@ void InetLayer::HandleTCPInactivityTimer(chip::System::Layer * aSystemLayer, voi
     InetLayer & lInetLayer = *reinterpret_cast<InetLayer *>(aAppState);
     bool lTimerRequired    = lInetLayer.IsIdleTimerRunning();
 
-    for (size_t i = 0; i < INET_CONFIG_NUM_TCP_ENDPOINTS; i++)
-    {
-        TCPEndPoint * lEndPoint = TCPEndPoint::sPool.Get(*aSystemLayer, i);
-
-        if (lEndPoint == nullptr)
-            continue;
+    TCPEndPoint::sPool.ForEachActiveObject([&](TCPEndPoint * lEndPoint) {
```

Comment:
Please:

- Use explicit capture by reference of just `lEndpoint`
- Make explicit the return type of the lambda (` -> bool`) so that the `return true` inner is more obvious. It's not necessarily obvious to the reader who has never seen `ForEachActiveObject` API what the return value means (e.g. that `return true` means continue iterating.

Reply:
Acked, this change needs to apply to all other occurrence of ForEachActiveObject throughout the stack. https://github.com/project-chip/connectedhomeip/issues/9209 to follow up this refactor.